documentation/jetty/modules/programming-guide/pages/arch/threads.adoc

```diff
@@ -166,7 +163,7 @@ The modes of task consumption discussed above are captured by the `org.eclipse.j
 * `org.eclipse.jetty.util.thread.strategy.ProduceExecuteConsume` for the `Produce-Execute-Consume` mode.
 * `org.eclipse.jetty.util.thread.strategy.ExecuteProduceConsume` for the `Execute-Produce-Consume` mode.
 
-There is an additional implementation of `ExecutionStrategy`, namely `org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy`, that takes into account the `InvocationType` of the task when the task implements `Invocable` (otherwise it assumes `BLOCKING`).
+There is an additional implementation of `ExecutionStrategy`, namely `org.eclipse.jetty.util.thread.strategy.AdaptiveExecutionStrategy`, that takes into account both the availability of idle threads and the `InvocationType` of the task when the task implements `Invocable` (otherwise it assumes `BLOCKING`).
```

Comment:
`both the availability of idle threads and` looks like an implementation detail to me.